src/components/popup.tsx

```diff
@@ -39,11 +41,16 @@ export interface PopupProps {
 export interface PopupState {
     primaryBtnShowSpinner: boolean;
     secondaryBtnShowSpinner: boolean;
+    /** Saved solely to be used as a comparison in getDerivedStateFromProps to check whether the step prop has changed */
     storedStep: number;
 }
 
 export class Popup extends React.Component<PopupProps, PopupState> {
 
+    static defaultProps = {
+        width: 'sm'
+    }
```

Comment:
missing `;`

Reply:
it's a regular object, so you can't add a semi-colon

Reply:
you most definitely can: https://github.com/Ultimaker/react-web-components/blob/60094b1a5d8b2942160a4ac7c3cf41989df0eaa0/src/components/input_fields/input_field_wrapper.tsx#L72

Reply:
oh, I misunderstood what you meant